week1/homework/src/ContactList.js

```diff
@@ -0,0 +1,49 @@
+import Contact from "./Contact";
+
+export default class ContactList {
+  constructor() {
+    this._contacts = [];
```

Comment:
you are using both `this.contacts` and `this._contacts`

Reply:
@godspeedelbow Unfortunately "choose one" it's not a good feedback.
Here is why:
@jawkha In your specific case there is no difference because the implementation of the getter is simple, you are not doing much ... but ... I would have done the opposite, I'd have use the getter ( `this.contacts` ) because if the implementation of the getter changes you would take advantage of the operations in the getter across your public methods in your class.

Bypassing the getter (or setter) could be potentially be valid but it can invalidate your data. 

A good example of bypassing a getter would if data need to be represented differently of your internal variable... so better use the internal one... (in that case I'd say that getter was a bad choice). I'll bring an example of both situations during the next class.

Considering that JS it's a bit too flexible, I'd tend to prefer to use the getter.